news/2020-04-05-project-loved-april-2020.md

```diff
@@ -0,0 +1,194 @@
+
+## <a id="catch"></a>osu!catch
+
+osu!catch Loved candidates were chosen by [Sartan](https://osu.ppy.sh/users/4100941), [Tenshichan](https://osu.ppy.sh/users/1101600), [Wesley](https://osu.ppy.sh/users/2407265) and [Yumeno Himiko](https://osu.ppy.sh/users/1806962)!
+
+---
+
+[![](/wiki/shared/news/2020-04-05-project-loved-april-2020/catch/1-calameme-fortune.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[RedScreen - Calameme Fortune](https://osu.ppy.sh/beatmapsets/635412#fruits) by [Bunnrei](https://osu.ppy.sh/users/829284)  
+200 BPM, 2:18 | 7.87★  
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+Starting off the Loved rotation, we have a pretty famous song in the osu!catch community, *Calameme Fortune*. Wait, Cala"meme" Fortune? Yeah that's right, a remix of the well known *Calamity Fortune*. The star rating of the map is "only" 7.87, but don't let that fool you. There are a bunch of different patterns coming through, varying from normal streams to wiggles and stairs. The high approach rate with the BPM really compliments the variety of patterns since it requires quite some speed to keep up with some of them. Can you keep up with the map while still holding a combo?
+
+[![](/wiki/shared/news/2020-04-05-project-loved-april-2020/catch/2-preserved-valkyria.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[penoneri - Preserved Valkyria](https://osu.ppy.sh/beatmapsets/420533#fruits) by [Kyuare](https://osu.ppy.sh/users/2705312), [Hareimu](https://osu.ppy.sh/users/4138746), [Lacrima](https://osu.ppy.sh/users/4915649) and [Nelly](https://osu.ppy.sh/users/4741164)  
+155 BPM, 1:59 | 5.90★, 3.26★, 4.49★, 2.23★  
+*description written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+Would you look at that: four osu!catch maps in this round of Loved! Thanks to this, we were able to play around more with the type of maps we could pick, and vary more than usual. Hence why I am hereby bringing you a nice and calm Kyuare mapset. *Preserved Valkyria* has four difficulties, ranging from a Salad to an Overdose, which enables players from all levels to check out this map. There is also plenty of room to play around with mods on this map for the more advanced players. Back when this mapset was released, it was quite a popular pick in multiplayer lobbies, but that hype has since died out; the map is four years old after all. The quality of the map is still top notch though, Kyuare mapping penoneri has always been a treat. Anyway, I have preserved this valkyria long enough from you already, there is plenty to discover in this mapset. Go check it out!
```

Comment:
```suggestion
Would you look at that: four osu!catch maps in this round of Loved! Thanks to this, we were able to play around more with the type of maps we could pick, and vary more than usual. Hence why I am hereby bringing you a nice and calm Kyuare mapset. *Preserved Valkyria* has four difficulties, ranging from a Salad to an Overdose, which enables players from all levels to check out this map. There is also plenty of room to play around with mods on this map for the more advanced players. Back when this mapset was released, it was quite a popular pick in multiplayer lobbies, but that hype has since died out; the map is four years old after all. The quality of the map is still top-notch though, Kyuare mapping penoneri has always been a treat. Anyway, I have preserved this valkyria long enough from you already, there is plenty to discover in this mapset. Go check it out!
```